### PR TITLE
Avoid NRE in ImportRemoteInstanceController.import()

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs [hunk 1]
```diff
@@ -43,9 +43,14 @@ public async Task<IActionResult> Import(ImportViewModel model)
 
             var remoteClient = remoteClientList.RemoteClients.FirstOrDefault(x => x.ClientName == model.ClientName);
 
+            if (remoteClient == null)
+            {
+                return StatusCode((int)HttpStatusCode.BadRequest, "The Api Key was not recognized");
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs:48]:
```suggestion
                return StatusCode((int)HttpStatusCode.BadRequest, "The remote client was not provided");
```

**Review by Skrypt — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs [hunk 2]
```diff
@@ -43,9 +43,14 @@ public async Task<IActionResult> Import(ImportViewModel model)
 
             var remoteClient = remoteClientList.RemoteClients.FirstOrDefault(x => x.ClientName == model.ClientName);
 
+            if (remoteClient == null)
+            {
+                return StatusCode((int)HttpStatusCode.BadRequest, "The remote client was not provided");
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs:48]: Just for info the following does the same, yes better to not do it here for consistency with other lines of code below, but maybe a good canditate for a separate PR ;)

                return BadRequest("The remote client was not provided");

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs:48]: I will try to check the entire code to see if we have such thing .. thanks

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs:48]: #12344

**Pull request merged**